src/Shell/Shell.php

```diff
@@ -63,12 +63,16 @@ public function __construct(
      * ```php
      * $this->shell->execute('/bin/bash -c "set -o pipefail; firstCommand | secondCommand"');
      * ```
+     *
+     *  `commandline` should be always a string as symfony/process package v2.x doesn't support array-type `commandLine`
      */
     public function execute(string $command, array $args = []): ProcessInterface
     {
         try {
+            $args = array_map('escapeshellarg', array_filter($args));
```

Comment:
why _array_filter()_ is used here?

Reply:
I've removed `array_filter()` as argument can be `false` or `0`